frontend/components/States.tsx

```diff
@@ -0,0 +1,11 @@
+export let headerStates = {
+    "title": "title",
+    "search": "search",
+    "total": "total"
+}
+
+export let bottomStates = {
+    "none": "none",
+    "itemSelection": "itemSelection",
+    "total": "total"
+}
```

Comment:
Also, since there's no JSX in this file, you can change the file ending to `.ts`